doc/project/checklist.md

```diff
@@ -496,6 +496,15 @@ The V3 checklist has been reviewed to understand the scope and estimate effort.
 
 ## V2S
 
+### SEC_CM_TESTPLAN_COMPLETED
+
+The testplan has been updated with the necessary testpoints and covergroups to adequately verify all security countermeasures implemented in the DUT.
+These countermeasures are listed in the comportable IP Hjson file located at `hw/ip/<ip>/data/<ip>.hjson` (or equivalent).
+
+On OpenTitan, a security countermeasures testplan is auto-generated (the first time) by the `reggen` tool for each DUT, and is placed at `hw/ip/<ip>/data/<ip>_sec_cm_testplan.hjson` (or equivalent).
+This testplan has been imported into the main testplan written for the DUT.
+Tests implemented to verify the security countermeasures have been mapped to these testpoints.
```

Comment:
I am not sure what is meant by this? should this have been :
```suggestion
Coverpoints implemented to verify the security countermeasures have been mapped to these test points.
```

Reply:
I think that "testpoint" is a term of art and (here) means "entries in the testplan". So I think Sri's text makes sense.

Of course, I also think that a competent DV engineer will write coverpoints to make sure that his/her proposed test does actually tickle the countermeasure in question.

Reply:
That is correct.